src/PSDMatrices.jl

```diff
@@ -40,6 +40,10 @@ function X_A_Xt(; A::PSDMatrix, X::AbstractMatrix)
     return PSDMatrix(A.R * X')
 end
 X_A_Xt(A::PSDMatrix, X::AbstractMatrix) = X_A_Xt(A=A, X=X)
+function X_A_Xt!(out::PSDMatrix, A::PSDMatrix, X::AbstractMatrix)
+    mul!(out.R, A.R, X')
+    return out
+end
```

Comment:
should we adopt the same keyword-style argument policy for this function as well? at least regarding the A and X? I think this might help the consistency and usability. WDYT?